src/function/pragma/pragma_queries.cpp

```diff
@@ -15,7 +16,16 @@ string PragmaTableInfo(ClientContext &context, const FunctionParameters &paramet
 }
 
 string PragmaShowTables(ClientContext &context, const FunctionParameters &parameters) {
-	return "SELECT name FROM sqlite_master ORDER BY name;";
+	auto catalog = DatabaseManager::GetDefaultDatabase(context);
+	auto table_query = "SELECT table_name as name FROM duckdb_tables where database_name='" + catalog + "'";
+	auto view_query = "SELECT view_name as name FROM duckdb_views where database_name='" + catalog + "'";
+	auto index_query = "SELECT index_name as name FROM duckdb_indexes where database_name='" + catalog + "'";
+
+	auto all_query = "select name from tables union all select name from views union all select name from indexes";
+
+	auto pragma_query = "with tables as ( " + table_query + "), views as ( " + view_query + +" ), indexes as ( " +
```

Comment:
`view_query + +` that's a typo, no?
Could you please change the string creation in this PR to use StringUtil::Format?

You'll need to include `duckdb/common/string_util.hpp`

Reply:
👍 Thanks for the comment that will indeed make code cleaner. Will update.

Reply:
Updated